contracts/Gateway.sol

```diff
@@ -40,7 +41,7 @@ contract Gateway is ProtocolVersioned, Owned {
     /** Events */
 
     /** Below event is emitted after successful execution of requestStake */
-    event StakeRequested(address _staker, uint256 _amount, address _beneficiary);
+    event StakeRequested(address _staker, uint256 _amount, uint256 _stakedAmount, address _beneficiary);
```

Comment:
What is the difference between `_amount` and `_stakedAmount`? It would be clearer if `_amount` had a more more descriptive name like `_requestedAmount`.